db/seeds/source_types.yml

```diff
@@ -264,6 +264,20 @@ openshift:
           :is: true
   :vendor: Red Hat
   :icon_url: "/apps/frontend-assets/platform-logos/openshift-container-platform.svg"
+rh-marketplace:
+  :product_name: Red Hat Marketplace
+  :vendor: Red Hat
```

Comment:
Currently, there are two categories in the ui: `Red Hat` and `Cloud` sources. 

Red Hat Source: `vendor == Red Hat`
Cloud Source: `vendor != Red Hat`

With introduction of Marketplace sources this is no longer possible to use - I think we could introduce a new attribute, something like `category`, to distinguish them.

cc @lindgrenj6

Reply:
(Maybe `type` would be better word.)

Reply:
So do you mean to have `Vendor` **:** `Cloud`, `Red Hat` or `Marketplace` ? 🤔 

It could work, but it is a breaking change if anyone is using the information now and we would need at least to coordinate UI/API deploy because it filters by vendor to show selection on Cloud/Red Hat section of the app.

Reply:
Okay... so I'm unsure what to do here. Do we finally change `vendor` to `type`, or do we leave it as it is? For now it seems that the user will be able to add a single Marketplace source for the API token, and that's it...

Reply:
So for now I'm thinking we should just leave it as Vendor - I know with marketplace it _technically_ isn't a Red Hat source but it can live there for now. 

We will also be needing to add certain logic for how we handle the marketplace apikey authentications (validations for only one per user, making sure that user has the right perms, etc) since they're a completely different usecase than we've had before. We can do all of that in a followup since its going to be a breaking change.

Reply:
Yeah, we can go on and merge it. It should not be a blocker, unless we are going to prod. It could get messy in the UI. 😅 

I guess we should update the disabled source types array to hide it in prod (possibly in stage too? 🤔 )